composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "beans/beans",
```

Comment:
I'm unsure what vendor name you want here.

Reply:
We have multiple names:

- beans 
- tm-beans 
- getbeans 

Let’s standardize on beans.  

Therefore, let’s stick with vendor name of `beans` and the package name of `beans`.